src/openzaak/components/catalogi/admin/admin_views.py

```diff
@@ -64,13 +66,37 @@ def post(self, request, *args, **kwargs):
             else:
                 try:
                     with transaction.atomic():
-                        import_zaaktype_for_catalogus(
+                        (
+                            files_found,
+                            files_not_found,
+                            files_received,
+                        ) = import_zaaktype_for_catalogus(
                             catalogus_pk, request.session["file_content"], {}, {}
                         )
-
-                    messages.add_message(
-                        request, messages.SUCCESS, _("ZaakType successfully imported")
-                    )
+                    if len(files_found) < 1:
+                        messages.add_message(
+                            request,
+                            messages.WARNING,
+                            mark_safe(
+                                _(
+                                    "No files found. Expected: {} but received:<br>{}"
+                                ).format(
+                                    ", ".join(files_not_found),
+                                    ", ".join(
+                                        [
+                                            f"<b>{escape(file)}</b>"
+                                            for file in files_received
+                                        ]
+                                    ),
+                                ),
+                            ),
+                        )
```

Comment:
Instead of `str.format` and `mark_safe` you can use `format_html`
I'm also not sure that `str.format` and `mark_safe` work with lazy translations
```suggestion
                        msg = _("No files found. Expected: {files_not_found} but received: {files_received}")
                        msg_dict = {"files_found": ", ".join(files_not_found), "files_received": ". ".join(files_received)}
                        messages.add_message(request, messages.WARNING, format_html(msg, **msg_dict))

```

Reply:
This is done.